asrtoolkit/data_structures/corpus.py

```diff
@@ -57,6 +58,35 @@ def validate(self):
 
         return valid
 
+    def prepare_for_training(self, target, sample_rate=16000, nested=False):
+        """
+        Prepare one exemplar for training
+        Returning a new exemplar object with updated file locations
+        and a resampled audio_file
+        """
+        if nested:
+            af_target_file = os.sep.join(target, "sph",
+                                         basename(self.audio_file.location))
+            tf_target_file = os.sep.join(
+                target, "stm", basename(self.transcript_file.location))
+        else:
+            af_target_file = os.sep.join(target,
+                                         basename(self.audio_file.location))
+            tf_target_file = os.sep.join(
```

Comment:
Maybe use `os.path.join` instead of `os.sep.join` here? It's safer than `os.sep.join`, particularly on [windows](https://stackoverflow.com/questions/16789714/differences-between-use-of-os-path-join-and-os-sep-concatenation).